src/main/kotlin/no/nav/tiltaksarrangor/controller/response/UlestEndringResponse.kt

```diff
@@ -49,5 +66,25 @@ fun List<UlestEndring>.toResponse(
 				it.oppdatering.forslag.toResponse(arrangornavn, ansatte, enheter),
 			),
 		)
+		is Oppdatering.NavBrukerEndring -> UlestEndringResponse(
+			it.id,
```

Comment:
nit: Kan være lurt å navngi parametre når det er flere av samme type etter hverandre for å unngå problemer med rekkefølgen